.changes/next-release/bugfix-s3-90647.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "bugfix",
+  "category": "s3",
+  "description": "Disable usage of mb command with S3 Express directory buckets."
```

Comment:
Let's make the `description` consistent between the `develop` and `v2` branches.

Reply:
Ok, I updated the description on the v2 branch to match this one.